src/main/java/hudson/security/AuthorizationMatrixProperty.java

```diff
@@ -243,21 +252,21 @@ public boolean hasPermission(String sid, Permission p) {
             if (!p.getEnabled()) {
                 continue;
             }
-            Set<String> set = grantedPermissions.get(p);
+			Set<String> set = grantedPermissions.get(p);
```

Comment:
revert indentation changes? Or if not, make the indentation actually consistent.